hack/e2e-suite/guestbook.sh

```diff
@@ -37,11 +37,13 @@ GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
 
 function teardown() {
   ${KUBECTL} stop -f "${GUESTBOOK}"
-  if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
+  if [[ "${KUBERNETES_PROVIDER}" == "gce" ]] || [[ "${KUBERNETES_PROVIDER}" == "gke" ]]; then
     local REGION=${ZONE%-*}
-    gcloud compute forwarding-rules delete -q --region ${REGION} "${INSTANCE_PREFIX}-default-frontend" || true
-    gcloud compute target-pools delete -q --region ${REGION} "${INSTANCE_PREFIX}-default-frontend" || true
-    gcloud compute firewall-rules delete guestbook-e2e-minion-8000 -q || true
+    if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
+      gcloud compute forwarding-rules delete -q --project="${PROJECT}" --region ${REGION} "${INSTANCE_PREFIX}-default-frontend" || true
```

Comment:
It seems like we'd leak the forwarding rule/target pool on GKE without some cleanup here.